test/utils/net.go

```diff
@@ -64,7 +68,13 @@ func PlatformPorts(includePublicPorts bool) (ports []string) {
 		ServicePort("vault"),
 		ServicePort("consul"),
 		ServicePort("redis"),
+		ServicePort("security-proxy-auth"),
```

Comment:
Since nginx(HTTP) uses a private port, it is necessary to include it as part of the platform's ports:
```suggestion
                ServicePort("nginx(http)"),
		ServicePort("security-proxy-auth"),
		
```